app/src/org/commcare/views/QuestionsView.java

```diff
@@ -139,7 +143,7 @@ public QuestionsView(Context context, FormEntryPrompt[] questionPrompts,
             widgets.add(qw);
             mView.addView(qw, mLayout);
             
-            qw.setChangedListener(this);
+            qw.setChangedListeners(this, blockingActionsManager);
```

Comment:
Moved this to the question widget centrally as part of the listener setup (which is what connects it "to the screen" more or less, so should be the same path), lets us access it broadly from each widget.